examples/tls.rs

```diff
@@ -86,16 +86,14 @@ async fn main() -> Result<()> {
         .await?;
 
     // Rows can be parsed as tuples
-    if let Some(rows) = session
+    let result = session
         .query("SELECT a, b, c FROM examples_ks.tls", &[])
-        .await?
-        .rows
-    {
-        for row in rows.into_typed::<(i32, i32, String)>() {
-            let (a, b, c) = row?;
-            println!("a, b, c: {}, {}, {}", a, b, c);
-        }
+        .await?;
+    let mut iter = result.rows_typed::<(i32, i32, String)>()?;
+    while let Some((a, b, c)) = iter.next().transpose()? {
+        println!("a, b, c: {}, {}, {}", a, b, c);
     }
+
```

Comment:
ditto about behavior, not going to post more comments about this as there are just too many examples